<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool_reboot</class>
 <widget class="QMainWindow" name="tool_reboot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>定时重启</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>84</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>自动重启</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>时间设置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>288</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator:checked
{
	border-image:url(:/png/shuncom_ico/png_ico/单选框.png)
}
</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>关闭</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>51</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>开启</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: rgb(50, 50, 51);
	
	background-color: rgb(255, 255, 255);
	border : none;
}
QComboBox::down-arrow{
	image: url(:/png/shuncom_ico/png_ico/下拉4.png);
	hight : 16px;
	width : 16px;
	
}
QComboBox::drop-down{
	border-style: none;
}</string>
        </property>
        <item>
         <property name="text">
          <string>每天</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期三</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期四</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期五</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期六</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>星期天</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTimeEdit" name="timeEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QTimeEdit
{
	
	color: rgb(50, 50, 51);
	border-radius:1px;
	
	
}
QTimeEdit::up-button
{
    image:none;
}


QTimeEdit::down-button
{
    
	image:none;
}
</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="keyboardTracking">
            <bool>true</bool>
           </property>
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/png/shuncom_ico/png_ico/时钟.png);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>157</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 98, 102);</string>
       </property>
       <property name="text">
        <string>已选择设备数:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{


    /*字体颜色*/ 
	color: rgb(255, 255, 255);
 	/*背景颜色*/  
	background-color: rgb(21, 91, 212);
    /*边框圆角半径为8像素*/ 
    border-radius:0px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:10pt;
    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color:rgb(255,255,255);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../shuncom_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
